docs/DEVELOPING.md

```diff
@@ -12,6 +12,138 @@
 - [PostgreSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-postgresql.html)
 - [S3](https://www.elastic.co/guide/en/enterprise-search/master/connectors-s3.html)
 
+## Confluence Connector
+
+The [Elastic Confluence connector](../connectors/sources/confluence.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **build a connector** workflow. See [Connector clients and frameworks](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+Confluence versions 7 or later are compatible with Elastic connector frameworks.
```

Comment:
It will be more clear if we rephrase the compatibility section to be something like:

Confluence Cloud or Confluence Server/Data Center versions 7 or later are compatible with Elastic connector frameworks.